### ISSUE TITLE
Insert a table row not removing the source's style

### ISSUE THREAD

**Issue by helianthuswhite** [xdan/jodit#933]
**Jodit Version:**  3.24.2

**Browser:** Chrome  
**OS:** Mac  
**Is React App:** True 

**Code**
When i select a style cell and insert a row before it, i found the new cell inherit the source's style. I don't know it is ok by design because when i insert a column, i get a new cell without any more style. 

I think it should be consistent whether i insert a row or a column.

Maybe jodit should clear the style when somebody insert a new row? The code just like this:

```js
// A *self-contained* demonstration of the problem follows...
$$('td,th', row).forEach(cell => {
  cell.innerHTML = '';
  cell.removeAttribute('style');
});
```
<img width="655" alt="image" src="https://user-images.githubusercontent.com/12878076/212631094-9d84bed8-2262-4456-91bc-d75750111c85.png">

**Expected behavior:**

**Actual behavior:**
![image](https://user-images.githubusercontent.com/12878076/212630953-e20757ab-0fb9-48d0-85b0-69e92c05fa9b.png)

**Comment by xdan** [comment]
Yes, it's so conceived by design, you can customize the entire row or column, and then when you put a new one, the styles should be the same. Other editors work in a similar way

**Comment by xdan** [comment]
Closing — triaged as expected behaviour/wontfix at the time, no activity since.
